### DISCUSSION TITLE
MKBHD Gave this project a shout-out

### DISCUSSION THREAD

**Discussion by Firecul** [ThioJoe/YT-Spammer-Purge#772] [General]
MKBHD did a video about YT spam and gave this project a mention. I expect we'll see another influx of people again.

https://www.youtube.com/watch?v=1Cw-vODp-8Y

**Comment by DinhHuy2010** [comment]
trolled? It is real

**Reply by KendallDoesCoding** [reply]
yeah, indeed it's real!

**Reply by Firecul** [reply]
> yeah, indeed it's real!

Yeah that was my fault, I had the wrong link when I posted this 🤦‍♂️

**Reply by KendallDoesCoding** [reply]
Oh ok.

**Comment by TechStudent10** [comment]
Not an April Fools joke. MKBHD did showcase this.

**Comment by ThioJoe** [comment]
Yep saw that

**Comment by TechStudent10** [comment]
> https://youtu.be/3y4-qqMgmwQ

That is a link to an unlisted video on your channel. The correct link is https://youtu.be/1Cw-vODp-8Y

Edit: removed `www` from URL

**Reply by Firecul** [reply]
FFS, that's my fault. Copying and pasting MKBHD's after using my link I must not have pressed the right button on my mouse. Thanks for pointing it out. Crazy no one checked before now though

**Reply by TechStudent10** [reply]
Hey no problem :)

**Comment by mendelsphotography** [comment]
The correct Link is https://www.youtube.com/watch?v=1Cw-vODp-8Y @TechStudent11 Does not work

**Reply by TechStudent10** [reply]
Yes I noticed thanks. It turns out that `www.` at the beginning of a shortened video link doesn't work.

**Reply by Firecul** [reply]
Yip, I'm an idiot confirmed

**Reply by mendelsphotography** [reply]
Oh and @Firecul We All Make mistakes does not make you an idot

**Reply by KendallDoesCoding** [reply]
Yeah, exactly!

**Comment by KendallDoesCoding** [comment]
This kinda is covered in #253 , there was no need of making a new discussion, but no worries :)!

**Reply by KendallDoesCoding** [reply]
but it's really cool that MKBHD made a video about this.
